client/grpc/client.go

```diff
@@ -0,0 +1,164 @@
+/*
+ * Flow Go SDK
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package grpc client provides a Go client for the Flow Access gRPC API.
+//
+// The Access API provides a set of methods that can be used to submit transactions
+// and read state from Flow. This client is compatible with the Access API implemented by the
+// Access Node role, as well as the mock Access API exposed by the Flow Emulator.
+//
+// The full Access API specification is here: https://docs.onflow.org/access-api/
+package grpc
+
+import (
+	"context"
+
+	"google.golang.org/grpc"
+
+	"github.com/onflow/cadence"
+	"github.com/onflow/flow-go-sdk"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+const EMULATOR_URL = "127.0.0.1:3569"
+const TESTNET_URL = "access.devnet.nodes.onflow.org:9000"
+const CANARYNET_URL = "access.canary.nodes.onflow.org:9000"
+const MAINNET_URL = "access.mainnet.nodes.onflow.org:9000"
```

Comment:
Mentioned above, but best to use [mixed caps](https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps) for these.